### PR TITLE
[loki-distributed] Increase default max memcached object size

### PR DESCRIPTION
**Pull request by AndreZiviani**
I was getting the following error:
```
level=error ts=2021-07-09T15:18:02.016545993Z caller=memcached.go:235 msg="failed to put to memcached" name=chunks err="server=172.23.11.216:11211: memcache: unexpected response line from \"set\": \"SERVER_ERROR object too large for cache\\r\\n\""
```

memcached limits the maximum size of an item to 1MB, this increases it to 32MB

Cortex helm chart has this [enabled by default](https://github.com/cortexproject/cortex-helm-chart/blob/cff890e2f374663c8ceedae1a989de1ab1a7053d/values.yaml#L1458)

Also fixes a bug on `README.md.gotmpl` introduced by a previous commit

### CODE REVIEW

File: charts/loki-distributed/README.md [hunk 1]
```diff
@@ -113,10 +114,9 @@ kubectl delete statefulset RELEASE_NAME-loki-distributed-querier -n LOKI_NAMESPA
 | gateway.image.registry | string | `"docker.io"` | The Docker registry for the gateway image |
 | gateway.image.repository | string | `"nginxinc/nginx-unprivileged"` | The gateway image repository |
 | gateway.image.tag | string | `"1.19-alpine"` | The gateway image tag |
-| gateway.ingress.annotations | object | `{}` | Annotations for the gateway ingress |
+| gateway.ingress.annotations | object | `{}` | Ingress Class Name. MAY be required for Kubernetes versions >= 1.18 ingressClassName: nginx -- Annotations for the gateway ingress |
 | gateway.ingress.enabled | bool | `false` | Specifies whether an ingress for the gateway should be created |
-| gateway.ingress.hosts | list | `[{"host":"gateway.loki.example.com","paths":[{"path": "/"}]}]` | Hosts configuration for the gateway ingress |
-| gateway.ingress.ingressClassName | string | `nil` | Ingress Class Name. MAY be required for Kubernetes versions >= 1.18 |
```
Comment (Whyeasy) [inline charts/loki-distributed/README.md:119]: @AndreZiviani It looks like something went wrong with the doc generation with helm-docs. Could you please double check this?

Comment (AndreZiviani) [reply charts/loki-distributed/README.md:119]: I'm not sure what you mean, if you are talking specifically about `ingressClassName` it was commented out on #522 but looks like the docs wasn't updated

Comment (Whyeasy) [reply charts/loki-distributed/README.md:119]: Yeah it seems you are right.

**Comment by Whyeasy** [comment]
Could you please upgrade the chart version so we can get this merged.

**Review by Whyeasy — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**